subgraph/src/indexes.ts

```diff
@@ -111,15 +111,93 @@ const computeUpdatedMorphoIndex = (
   return newIndex;
 };
 
+const computeUpdatedMorphoIndexV2 = (
+  marketAddress: Address,
+  blockTimestamp: BigInt,
+  lastMorphoIndex: BigInt,
+  lastUpdateBlockTimestamp: BigInt,
+  lastPercentSpeed: BigInt,
+  lastTotalScaled: BigInt,
+  marketSide: string
+): BigInt => {
+  const obj = ipfsJson();
+  // sync eventual previous epoch
+  const prevEpochId = timestampToEpochId(obj, lastUpdateBlockTimestamp);
+  const currentEpochId = timestampToEpochId(obj, blockTimestamp);
+  if (!currentEpochId) return lastMorphoIndex; // TODO: to move at the end
```

Comment:
Why should it be moved at the end?

Reply:
If not moved, there are unnecessary checks `currentEpochId` down below

Reply:
Yeah that was an outdated comment that is not true anymore